core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/codeGen/MarkersExtractor.kt

```diff
@@ -54,7 +54,7 @@ internal object MarkersExtractor {
 
     private fun getFields(markerClass: KClass<*>, nullableProperties: Boolean): List<GeneratedField> {
         val order = getPropertiesOrder(markerClass)
-        return markerClass.declaredMemberProperties.sortedBy { order[it.name] ?: Int.MAX_VALUE }.mapIndexed { _, it ->
```

Comment:
I wonder if it was a deliberate choice to reduce amount of generated code in Jupyter. 
could you check what is generated after executing the second cell?
```
val df = dataFrameOf("a")(123)
```

second cell
```
val df1 = df.add("b") { 321 }
```

Reply:
`Firstly, I need to make the first line a var to make isOpen = true` 
This sounds suprising. I don't think it was the case before, was it? 

`So, I suspect for Jupyter it gets the properties for extensions in yet another way.`
Oh, right. I think Jupyter works on DataFrame / ColumnSchema level and uses existing markers somehow.

Reply:
It is the case since https://github.com/Kotlin/dataframe/pull/193. Or well, it was the case but not everywhere and not consistently (Even the tests were inconsistent about it). Since that PR it's always consistent.